website/content/api-docs/operator/keyring.mdx

```diff
@@ -151,7 +151,7 @@ $ curl \
 ```
 
 
-[Key Management]: /docs/concepts/key-management
-[`nomad operator root keyring`]: /docs/commands/operator/root/keyring
+[Key Management]: /docs/operations/key-management
+[`nomad operator root keyring`]: /docs/commands/operator/keyring
```

Comment:
This is the right link, we're just missing an Overview page for `operator root`, which is associated with links like: https://www.nomadproject.io/docs/commands/operator/root/keyring-rotate

I can provide some copy for that.

Reply:
Thanks @tgross! feel free to push to this PR if/when you get a chance

(We're aiming to get these fixed by the [Developer / Nomad](https://developer.hashicorp.com/nomad) GA rollout tomorrow)